src/backend/impress/settings.py

```diff
@@ -462,7 +462,9 @@ class Base(Configuration):
     # Session
     SESSION_ENGINE = "django.contrib.sessions.backends.cache"
     SESSION_CACHE_ALIAS = "default"
-    SESSION_COOKIE_AGE = 60 * 60 * 12
+    SESSION_COOKIE_AGE = values.IntegerValue(
```

Comment:
```suggestion
    SESSION_COOKIE_AGE = values.PositiveIntegerValue(
```

This is a nitpick, you can decide to leave it as it.